tools/precompile/cldr_data.mjs

```diff
@@ -201,17 +211,11 @@ const loadAllKeymaps = async () => {
     const layout = await loadKeyboard(
       l.match("^(...?)-")[1],
       false,
-      path.join(getStaticPath(), "cldr/keyboards", os, l)
+      path.join("static/cldr/keyboards", os, l)
```

Comment:
Lift out "static/cldr/keyboards" to a const? I'm betting we'll need to change it when we migrate build systems.

Reply:
> > We still import cldr from the renderer process
> 
> Yeah, just the import is what causes trouble.

Ah, I see. I'll move it to main then.

> (Also, is that extract why clicking on the select for keyboard names takes ages?)

Nah, it takes ages because the data is ~1.8mb of json, and we're rendering that into a dropdown, including iterating  through the whole thing, and sorting it. We should pregenerate that, too.